examples/block_jid.py

```diff
@@ -114,6 +116,11 @@ def jid(s):
             help="If given, prints the block list at the end of the operation",
         )
 
+        config_default = os.path.expanduser(
+            '~/.config/aioxmpp_credentials.ini')
```

Comment:
Can we at least settle on ``~/.aioxmpprc``? There **is** ``$HOME`` directory on Windows (yes, it is in the wrong place, etc.).